### PR TITLE
Samuel

### PR DESCRIPTION
**Pull request by samuel408**
plz merge my code :) I added the information from the API

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -307,6 +307,9 @@ <h1>Ketchup on COVID-19</h1>
         </section>
     </div>
     <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.24.0/moment.min.js"></script>
+    <script src="../tbd/assets/js/script.js"></script>
```
Comment (yulduzetta) [inline index.html]: is not line #312 dupe of #313?

File: assets/js/script.js [hunk 2]
```diff
@@ -1,39 +1,60 @@
...
+            currentRecoveries.innerHTML = "Recovered Cases: " + data.data.recovered;
+
+            info.appendChild(countryName);
+            info.appendChild(lastChecked);
+            info.appendChild(currentCases);
+            info.appendChild(currentDeaths);
+            info.appendChild(currentRecoveries);
+
+
+
```
Comment (yulduzetta) [inline assets/js/script.js]: do we need these empty lines?

File: assets/js/script.js [hunk 3]
```diff
@@ -1,39 +1,74 @@
...
+          });
+        } else {
+          alert("Error: " + response.statusText);
+        }
+      })
+      .catch(function(error) {
+        alert("Unable to connect to GitHub");
+      });
```
Comment (yulduzetta) [inline assets/js/script.js]: could you take this entire logic into its own function? There is more code that will be defined in this event listener, and we'd rather keep it clean and modular. Thanks

File: assets/js/script.js [hunk 4]
```diff
@@ -1,80 +1,127 @@
...
+           displayResults(data);
+          
+         });
+       } else {
+         alert("Error: " + response.statusText);
+       }
+     })
+     .catch(function(error) {
+       alert("Unable to connect to GitHub");
+     });
```
Comment (yulduzetta) [inline assets/js/script.js:44]: could you move these lines inside `displayResults() `

File: assets/js/script.js [hunk 5]
```diff
@@ -1,80 +1,127 @@
+
 /*TODO : when search button is clicked we 
 get the users import and ad it to the API to get the response
 for the specific place they inserted
 */
+//date for the api it only takes format YYYY/MM/DD
+
+var date = moment(). format("YYYY-MM-DD");
+console.log(date)
```
Comment (yulduzetta) [inline assets/js/script.js:9]: can we remove this console.log statement

File: assets/js/script.js [hunk 6]
```diff
@@ -1,80 +1,127 @@
...
+            //displays results
+           displayResults(data);
+          
+         });
+       } else {
+         alert("Error: " + response.statusText);
+       }
+     })
+     .catch(function(error) {
+       alert("Unable to connect to GitHub");
```
Comment (yulduzetta) [inline assets/js/script.js:43]: no native alerts are allowed in our project, let's remove them

File: assets/js/script.js [hunk 7]
```diff
@@ -1,80 +1,127 @@
...
+     });
 });
-// displays covid results.
-function displayRates(data) {}
+//function that displays  logic
+function displayResults(data){
+    if (data.data.location === "Global"){
+        errorMessage.innerHTML = "This country is not yet supported by our program, please have a look at the global stats!";
```
Comment (yulduzetta) [inline assets/js/script.js:49]: 👍

**Comment by samuel408** [comment]
merged my code with yulduz 's pull

**Pull request closed without merging**